From ced502801faa24c4e8791a396cbc99ad6290779c Mon Sep 17 00:00:00 2001
From: Michael Stack <stack@apache.org>
Date: Thu, 1 Feb 2018 08:24:54 -0800
Subject: [PATCH] HBASE-19911 Convert some tests from small to medium because
 they are timing out: TestNettyRpcServer, TestClientClusterStatus; ADDENDUM2
 Up timeout for smalltests from 30 seconds to 60 seconds -- a pause on jenkins
 can mess up smalltests

---
 .../src/test/java/org/apache/hadoop/hbase/HBaseClassTestRule.java     | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/hbase-common/src/test/java/org/apache/hadoop/hbase/HBaseClassTestRule.java b/hbase-common/src/test/java/org/apache/hadoop/hbase/HBaseClassTestRule.java
index 74bd70e999..734ce3f241 100644
--- a/hbase-common/src/test/java/org/apache/hadoop/hbase/HBaseClassTestRule.java
+++ b/hbase-common/src/test/java/org/apache/hadoop/hbase/HBaseClassTestRule.java
@@ -63,7 +63,9 @@ public final class HBaseClassTestRule implements TestRule {
     for (Class<?> c : categories[0].value()) {
       if (c == SmallTests.class) {
         // See SmallTests. Supposed to run 15 seconds.
-        return 30;
+        // Lots of these timeout on Jenkins... a stall of ten or twenty seconds mess up what looks
+        // fine when run local.
+        return 60;
       } else if (c == MediumTests.class) {
         // See MediumTests. Supposed to run 50 seconds.
         return 180;
-- 
2.11.0 (Apple Git-81)

